setup/install.py

```diff
@@ -379,79 +339,75 @@ def main():
-        venv_check()
-
-        # Import packages that depend on pattoo shared
-        from _pattoo import configure
-        from pattoo_shared.installation import packages, systemd, environment
-
-        # Set up essentials for creating the virtualenv
-        pattoo_home = get_pattoo_home()
-        venv_dir = os.path.join(pattoo_home, 'pattoo-venv')
-        if getpass.getuser() != 'travis':
-            environment.environment_setup(venv_dir)
-        venv_interpreter = os.path.join(venv_dir, 'bin/python3')
-        installation_dir = '{} {}'.format(venv_interpreter, ROOT_DIR)
-
-        # Installs all pattoo components
-        if args.qualifier == 'all':
-            print('Installing everything')
-            configure.install(pattoo_home)
-            packages.install(ROOT_DIR, venv_dir, args.verbose)
-
-            # Import db after pip3 packages are installed
-            from _pattoo import db
-            db.install()
-            if shared.root_check() is True:
-                systemd.install(daemon_list=daemon_list,
-                                template_dir=template_dir,
-                                installation_dir=installation_dir,
-                                verbose=args.verbose)
-
-        # Configures pattoo and sets up database tables
-        elif args.qualifier == 'database':
-            print('Installing database tables')
-            configure.install(pattoo_home)
-            packages.install(ROOT_DIR, venv_dir)
-            # Import db after pip3 packages are installed
-            from _pattoo import db
-            db.install()
-
-        # Installs and starts system daemons
-        elif args.qualifier == 'systemd':
-            print('Installing systemd daemons')
-            if getpass.getuser() == 'root':
-                systemd.install(daemon_list=daemon_list,
-                                template_dir=template_dir,
-                                installation_dir=installation_dir,
-                                verbose=True)
-            else:
-                shared.log('You need to be running as root to install the daemons')
-
-        elif args.qualifier == 'pip':
-            # Installs additionally required pip3 packages
-            packages.install(ROOT_DIR, venv_dir, args.verbose)
-
-        # Sets up the configuration for pattoo
-        elif args.qualifier == 'configuration':
-            configure.install(pattoo_home)
-
-        # Print help if no argument options were triggered
+    # Perform checks
+    checks.parser_check(_parser.args()[1], _parser.args()[0])
+    checks.pattoo_shared_check()
+    checks.venv_check()
+
+    # Import packages that depend on pattoo shared
+    from _pattoo import configure
+    from pattoo_shared.installation import packages, systemd, environment
+
+    # Set up essentials for creating the virtualenv
+    pattoo_home = get_pattoo_home()
+    venv_dir = os.path.join(pattoo_home, 'pattoo-venv')
+    if getpass.getuser() != 'travis':
```

Comment:
Before this PR, I tried installing as a regular user in my `pattoo` repo directory `/opt/pattoo/pattoo`. I noticed that the `venv` was installed in `/opt/pattoo/pattoo/.pattoo-unittests/pattoo-venv` versus in my home directory like `$HOME/.pattoo-unittests/pattoo-venv`. The `venv` should not be installed within the code base.

Has this bee fixed in this PR?

Reply:
Yup, it has been fixed. I think that error would occur if your `$HOME` environment variable is set to `/opt/pattoo`. 
Normally it should install to the same directory as the directory for the unittest config files.
I'll create a unittest for this now.

But in the code segment below, `unittest_dir` is the location that the venv will be created in.


```
unittest_config_dir = (
        '{}{}{}'.format(os.environ['HOME'], os.sep, config_suffix))
    print('Setting config directory to {}'.format(unittest_config_dir))

    # Sets PATTOO_CONFIGDIR environment varaible to the unittest config dir
    if 'unittest' not in os.environ['PATTOO_CONFIGDIR']:
        os.environ['PATTOO_CONFIGDIR'] = unittest_config_dir

    unittest_dir = config_suffix.split(os.sep)[0]
    return unittest_dir
```

Reply:
wait, my bad. I see the problem now.